lib/gli/commands/scaffold.rb

```diff
@@ -121,7 +121,7 @@ def self.mk_rakefile(root_dir,dry_run,project_name,create_test_dir)
   rd.title = 'Your application title'
 end
 
-spec = eval(File.read('#{project_name}.gemspec'))
+spec = Gem::Specification.load("#{project_name}.gemspec")
```

Comment:
Bug in Ruby 3.3 - not sure why `eval(File.read(...))` didn't work but `Gem::Specification.load` seems to be the more correct way to do this.